Formula/nspr.rb

```diff
@@ -30,8 +30,11 @@ def install
         --enable-64bit
       ]
       system "./configure", *args
-      # Remove the broken (for anyone but Firefox) install_name
-      inreplace "config/autoconf.mk", "-install_name @executable_path/$@ ", "-install_name #{lib}/$@ "
```

Comment:
Does this actually break anything on Linux?

It involves `-install_name` which doesn't work on Linux, so this probably involves a code path that's reached only on macOS.

Reply:
Yes, the in inreplace fails with:

```
config.status: executing default commands
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
Utils::Inreplace::Error: inreplace failed
config/autoconf.mk:
  expected replacement of "-install_name @executable_path/$@ " with "-install_name /home/linuxbrew/.linuxbrew/Cellar/nspr/4.31/lib/$@ "
```